lampo-core-wallet/src/lib.rs

```diff
@@ -72,7 +72,7 @@ impl CoreWalletManager {
         xprv: lampo_common::bitcoin::PrivateKey,
         channel_keys: Option<String>,
     ) -> error::Result<(bdk::Wallet, LampoKeys)> {
-        use bdk::bitcoin::bip32::ExtendedPrivKey;
+        use bdk::bitcoin::bip32::Xpriv;
```

Comment:
mh this should be added in a separate commit, like adding a commit to upgrade bdk and then add another commit to upgrade ldk

Reply:
Okay